apps/src/signUpFlow/AccountType.tsx

```diff
@@ -1,13 +1,20 @@
-import React from 'react';
+import React, {useState} from 'react';
 
+import Button from '@cdo/apps/componentLibrary/button/Button';
 import locale from '@cdo/apps/signup/locale';
 import AccountBanner from '@cdo/apps/templates/account/AccountBanner';
 
+import FontAwesomeV6Icon from '../componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon';
+import {Heading2} from '../componentLibrary/typography';
 import AccountCard from '../templates/account/AccountCard';
 
+import FreeCurriculumDialog from './FreeCurriculumDialog';
+
 import style from './accountType.module.scss';
 
 const AccountType: React.FunctionComponent = () => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
```

Comment:
tiny nit: could we make this more specific, ie `isFreeCurriculumModalOpen`? I think this will be the only modal on this page, so not a big deal, but if we ever added another, it could get confusing